api-reference/beta/toc.yml

```
@@ -11943,6 +11943,18 @@ items:
               href: api/samlorwsfedexternaldomainfederation-list-domains.md
             - name: Create domain
               href: api/samlorwsfedexternaldomainfederation-post-domains.md
+          - name: Federation with Azure AD
+            items:
+            - name: Federated domain configuration
+              href: resources/internaldomainfederation.md
+            - name: Get federated domain configuration
+              href: api/internaldomainfederation-get.md
+            - name: Create federated domain configuration
+              href: api/domain-post-federationconfiguration.md
+            - name: Update federated domain configuration
+              href: api/internaldomainfederation-update.md
+            - name: Delete federated domain configuration
+              href: api/internaldomainfederation-delete.md
```

Does calling it SAML or WS-Fed internal domain federation make sense and is it applicable for internal federation? I recall we discussed this with Namrata while doing the one for external identities where she advised that we explicitly call it SAML/WS-Fed because there were other protocols used by external IdPs. I wonder if the same can be said for internal IdP. https://docs.microsoft.com/en-us/graph/api/resources/samlorwsfedexternaldomainfederation

Suggestion to just use the verbs Get, Create etc is for scannability and because unlike for external IdP API docs, you don't have other methods that overlap with the basic CRUD methods.
```suggestion
          - name: SAML or WS-Fed internal domain federation
            href: resources/internaldomainfederation.md
            items:             
            - name: Get
              href: api/internaldomainfederation-get.md
            - name: Create
              href: api/domain-post-federationconfiguration.md
            - name: Update
              href: api/internaldomainfederation-update.md
            - name: Delete
              href: api/internaldomainfederation-delete.md
```